pkg/datapath/loader/loader.go

```diff
@@ -713,6 +725,12 @@ func endpointRewrites(ep datapath.EndpointConfiguration, lnc *datapath.LocalNode
 	cfg.EnableICMPRule = option.Config.EnableICMPRules
 	cfg.EnableLRP = option.Config.EnableLocalRedirectPolicy
 
+	if option.Config.DatapathMode != datapathOption.DatapathModeNetkit {
+		// only one of the enable_arp_* options must be true.
+		cfg.EnableArpPassthrough = ep.RequireARPPassthrough()
+		cfg.EnableArpResponder = !ep.RequireARPPassthrough()
+	}
+
```

Comment:
If the datapath mode is netkit, this part is skipped and both options are set to default value(0)